File: src/test/java/net/slipp/support/SlippEnvironmentTest.java
package net.slipp.support;

import static org.hamcrest.CoreMatchers.is;
import static org.junit.Assert.assertThat;

import org.junit.Test;

public class SlippEnvironmentTest {

	@Test
	public void loadProperties() {
		SlippEnvironment environment = new SlippEnvironment();
		assertThat(environment.getProperty("environment"), is("DEVELOPMENT"));
	}

}

File: src/test/scala/net/slipp/support/SlippEnvironmentTest.scala
package net.slipp.support

import org.hamcrest.CoreMatchers.is
import org.junit.Assert.assertThat
import org.junit.Test

class SlippEnvironmentTest {
  @Test def loadProperties {
    val environment: SlippEnvironment = new SlippEnvironment
    assertThat(environment.getProperty("environment"), is("DEVELOPMENT"))
  }
}
